crates/aptos-crypto/src/hash.rs

```diff
@@ -226,6 +226,11 @@ impl HashValue {
         self.hash[index]
     }
 
+    /// Return the shard id of this `HashValue`.
+    pub fn shard_id(&self) -> u8 {
```

Comment:
I'm not sure the notion of a shard ID belongs in the `aptos-crypto` crate... :-/

Reply:
Moved to storage/